train.py

```diff
@@ -21,11 +21,15 @@ def store_js(filename, data):
         f.write('export default ' + json.dumps(data, indent=2))
 
 BATCH_SIZE = 32
-NUM_EPOCHS = 40
+NUM_EPOCHS = 35
+HIDDEN_UNITS = 64
+MAX_INPUT_SEQ_LENGTH = 17
+MAX_TARGET_SEQ_LENGTH = 27
```

Comment:
You're defining hidden units, max input seq length, and max target seq length twice.